src/colourConverters.test.ts

```diff
@@ -0,0 +1,50 @@
+import {
+	hexToRGB,
+	isHexCode,
+	isNamedColour,
+	nameToHex,
+} from './colourConverters';
+
+describe('`isHexCode`', () => {
+	test('should return true given a valid hexcode', () => {
+		expect(isHexCode('#000')).toBe(true);
+		expect(isHexCode('#0101ff')).toBe(true);
```

Comment:
If `#` is truly optional, you should test against it!

```suggestion
		expect(isHexCode('#0101ff')).toBe(true);
		expect(isHexCode('123456')).toBe(true);
```

Reply:
`#` is now enforced, 👍